components/resources/speedreader_strings.grdp

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <grit-part>
-  <if expr="enable_speedreader">   
+  <if expr="enable_speedreader">
+    <message name="IDS_SPEEDREADER_BRAND_LABEL" desc="Used in places to show brand title">Brave Speedreader</message>
```

Comment:
nit: fix indenting on other lines to match this one, or either way make it consistent with rest of file